parsers/lib/AU_battery.py

```diff
@@ -1,31 +1,40 @@
 #!/usr/bin/python
 
-import requests
+"""Parser for South Australia's 129MWh battery built by Tesla."""
 
-#Parser for South Australia's 129MWh battery built by Tesla.
-#base_url gets generation status in 5 min intervals.
+import arrow
+import json
+import requests
 
-base_url = 'https://ausrealtimefueltype.global-roam.com/api/SeriesSnapshot?time='
+#nemlog_url gets generation status in 5 min intervals.
 
 def fetch_SA_battery():
     """
-    Makes a request to the NemWatch widget data source.
-    Finds dictionaries related to the South Australia battery.
-    Returns a float.
+    Makes a request to the nemlog api for South Australia battery data.
+    Returns a float or None.
     """
 
-    #Verify set to false to prevent SSL error due to domain mismatch.
-    req = requests.get(base_url, verify = False)
-    json_content = req.json()
-    dicts = json_content["seriesCollection"]
-    discharge_id = (item for item in dicts if item["id"] == "41").next()
-    storage_id = (item for item in dicts if item["id"] == "57").next()
+    today = arrow.now('Australia/Adelaide')
+    current = today.format('YYYYMMDD')
+    old = today.shift(days=-2).format('YYYYMMDD')
```

Comment:
why two days back?

Reply:
The default behavior of the api with no dates is to return data up to 2 days back.  I've kept that for the request but tbh as we only use the last value it doesn't make much difference.